lib/session/session_test.go

```diff
@@ -40,52 +52,47 @@ type SessionSuite struct {
 	clock clockwork.FakeClock
 }
 
-var _ = Suite(&SessionSuite{})
-
-func (s *SessionSuite) SetUpSuite(c *C) {
-	utils.InitLoggerForTests(testing.Verbose())
-}
-
-func (s *SessionSuite) SetUpTest(c *C) {
+func NewSessionSuite(t *testing.T) *SessionSuite {
```

Comment:
nit: I'm guessing this needn't be exported. Ditto for the suite type

Reply:
Adjusted